builddefs/common_features.mk

```diff
@@ -803,10 +806,7 @@ ifeq ($(strip $(UNICODE_COMMON)), yes)
 endif
 
 ifeq ($(strip $(PS2_MOUSE_ENABLE)), yes)
-    PS2_ENABLE := yes
-    MOUSE_ENABLE := yes
-    SRC += ps2_mouse.c
-    OPT_DEFS += -DPS2_MOUSE_ENABLE
+    $(call CATASTROPHIC_ERROR,PS2_MOUSE has migrated to pointing device,Please see docs/feature_pointing_device.md for more information.)
```

Comment:
Without updating the handful of boards that use ps2 mouse, this causes them to fail to compile.  

```
frobiac/blackbowl:default
handwired/108key_trackpoint:default
handwired/promethium:default
handwired/trackpoint:default
```

Reply:
These have now been ported over, hopefully they will compile in the QMK CI Build :crossed_fingers: